boards/arduino/opta/doc/index.rst

```diff
@@ -62,32 +87,41 @@ The current Zephyr arduino_opta_m4 board configuration supports the following ha
 
 Other hardware features are not yet supported on Zephyr porting.
 
-The default configuration per core can be found in the defconfig file:
-:zephyr_file:`boards/arduino/opta/arduino_opta_stm32h747xx_m4_defconfig`
+The default configuration per core can be found in the defconfig files:
+:zephyr_file:`boards/arduino/opta/arduino_opta_stm32h747xx_m4_defconfig` and
+:zephyr_file:`boards/arduino/opta/arduino_opta_stm32h747xx_m7_defconfig`.
 
 Pin Mapping
 ===========
 
-ARDUINO OPTA M4 has access to the 9 GPIO controllers. These controllers are responsible for pin muxing,
-input/output, pull-up, etc.
+Both the M7 and M4 cores have access to the 9 GPIO controllers. These
+controllers are responsible for pin muxing, input/output, pull-up, etc.
 
 For more details please refer to `ARDUINO-OPTA website`_.
 
 Default Zephyr Peripheral Mapping
 ---------------------------------
 
-- Status LED1 : PI0
-- Status LED2 : PI1
-- Status LED3 : PI3
-- Status LED4 : PH15
-- User button : PE4
+- Status LED1: PI0
+- Status LED2: PI1
+- Status LED3: PI3
+- Status LED4: PH15
+- Green "reset" LED: PH12
+- Red "reset" LED: PH11
+- Blue LED: PE5
+- User button: PE4
+- Relay 1: PI6
+- Relay 2: PI5
+- Relay 3: PI7
+- Relay 4: PI4
```

Comment:
@fogzot would be possible to add here the mapping of the inputs as I did in my PR?